mvn clean package: error, Could not find artifact javax.jnlp:jnlp:jar:1.6 at specified path /usr/lib/jvm/java-11-openjdk/lib/javaws.jar

```
Tests run: 191, Failures: 0, Errors: 0, Skipped: 0

[INFO] 
[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ delta-common ---
[INFO] Building jar: /home/guoyi/Downloads/open-delta/common/target/delta-common-1.0.3-SNAPSHOT.jar
[INFO] 
[INFO] ------------------< au.org.ala.delta:delta-ui-common >------------------
[INFO] Building DELTA User Interface Common 1.0.3-SNAPSHOT                [2/9]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Open DELTA suite 1.0.3-SNAPSHOT:
[INFO] 
[INFO] DELTA Common ....................................... SUCCESS [  6.292 s]
[INFO] DELTA User Interface Common ........................ FAILURE [  0.012 s]
[INFO] DIST ............................................... SKIPPED
[INFO] KEY ................................................ SKIPPED
[INFO] intkey ............................................. SKIPPED
[INFO] DELTA Editor ....................................... SKIPPED
[INFO] CONFOR ............................................. SKIPPED
[INFO] DELFOR ............................................. SKIPPED
[INFO] Open DELTA suite ................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  6.509 s
[INFO] Finished at: 2022-12-02T10:38:01+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project delta-ui-common: Could not resolve dependencies for project au.org.ala.delta:delta-ui-common:jar:1.0.3-SNAPSHOT: Could not find artifact javax.jnlp:jnlp:jar:1.6 at specified path /usr/lib/jvm/java-11-openjdk/lib/javaws.jar -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :delta-ui-common
```

--- a/confor/pom.xml
+++ b/confor/pom.xml
@@ -1,5 +1,5 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="https://maven.apache.org/POM/4.0.0" xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="https://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>au.org.ala.delta</groupId>
 	<artifactId>confor</artifactId>
@@ -7,9 +7,9 @@
 	<name>CONFOR</name>
 
 	<scm>
-		<connection>scm:svn:http://open-delta.googlecode.com/svn/trunk/</connection>
+		<connection>scm:svn:https://open-delta.googlecode.com/svn/trunk/</connection>
 		<developerConnection>scm:svn:https://open-delta.googlecode.com/svn/trunk/</developerConnection>
-		<url>http://code.google.com/p/open-delta/</url>
+		<url>https://code.google.com/p/open-delta/</url>
 	</scm>
 	<build>
 		<plugins>
@@ -77,8 +77,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>2.0.2</version>
 				<configuration>
-					<source>1.6</source>
-					<target>1.6</target>
+					<source>1.8</source>
+					<target>1.8</target>
 					<encoding>Cp1252</encoding>
 				</configuration>
 			</plugin>

--- a/delfor/pom.xml
+++ b/delfor/pom.xml
@@ -1,5 +1,5 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="https://maven.apache.org/POM/4.0.0" xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="https://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>au.org.ala.delta</groupId>
 	<artifactId>delfor</artifactId>
@@ -9,9 +9,9 @@
 
 
 	<scm>
-		<connection>scm:svn:http://open-delta.googlecode.com/svn/trunk/</connection>
+		<connection>scm:svn:https://open-delta.googlecode.com/svn/trunk/</connection>
 		<developerConnection>scm:svn:https://open-delta.googlecode.com/svn/trunk/</developerConnection>
-		<url>http://code.google.com/p/open-delta/</url>
+		<url>https://code.google.com/p/open-delta/</url>
 	</scm>
 	<build>
 
@@ -21,8 +21,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>2.0.2</version>
 				<configuration>
-					<source>1.6</source>
-					<target>1.6</target>
+					<source>1.8</source>
+					<target>1.8</target>
 					<encoding>Cp1252</encoding>
 				</configuration>
 			</plugin>

--- a/delta-editor/pom.xml
+++ b/delta-editor/pom.xml
@@ -1,5 +1,5 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="https://maven.apache.org/POM/4.0.0" xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="https://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>au.org.ala.delta</groupId>
 	<artifactId>delta-editor</artifactId>
@@ -9,18 +9,18 @@
 		<repository>
 			<id>maven2-repository.dev.java.net</id>
 			<name>Java.net Repository for Maven</name>
-			<url>http://download.java.net/maven/2/</url>
+			<url>https://download.java.net/maven/2/</url>
 			<layout>default</layout>
 		</repository>
 		<repository>
 			<id>shell-sourceforge-jp</id>
 			<name>maven repository on sourceforge.jp shell server</name>
-			<url>http://jfontchooser.sourceforge.jp/repository/say/swing/jfontchooser</url>		
+			<url>https://jfontchooser.sourceforge.jp/repository/say/swing/jfontchooser</url>		
 		</repository>
 	</repositories>
 
 	<scm>
-		<connection>scm:svn:http://open-delta.googlecode.com/svn/trunk/</connection>
+		<connection>scm:svn:https://open-delta.googlecode.com/svn/trunk/</connection>
 		<developerConnection>scm:svn:https://open-delta.googlecode.com/svn/trunk/</developerConnection>
 	</scm>
 	<build>
@@ -75,8 +75,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>2.0.2</version>
 				<configuration>
-					<source>1.6</source>
-					<target>1.6</target>
+					<source>1.8</source>
+					<target>1.8</target>
 					<encoding>Cp1252</encoding>
 				</configuration>
 			</plugin>

--- a/intkey/pom.xml
+++ b/intkey/pom.xml
@@ -1,5 +1,5 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="https://maven.apache.org/POM/4.0.0" xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="https://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>delta-intkey</groupId>
 	<artifactId>delta-intkey</artifactId>
@@ -9,25 +9,25 @@
 		<repository>
 			<id>maven2-repository.dev.java.net</id>
 			<name>Java.net Repository for Maven</name>
-			<url>http://download.java.net/maven/2/</url>
+			<url>https://download.java.net/maven/2/</url>
 			<layout>default</layout>
 		</repository>
 		<repository>
 			<id>shell-sourceforge-jp</id>
 			<name>maven repository on sourceforge.jp shell server</name>
-			<url>http://jfontchooser.sourceforge.jp/repository/say/swing/jfontchooser</url>
+			<url>https://jfontchooser.sourceforge.jp/repository/say/swing/jfontchooser</url>
 		</repository>
 		<repository>
 			<id>ala-repository</id>
 			<name>ala-repository</name>
-			<url>http://nexus.ala.org.au/content/groups/public/</url>
+			<url>https://nexus.ala.org.au/content/groups/public/</url>
 		</repository>
 	</repositories>
 
 	<scm>
-		<connection>scm:svn:http://open-delta.googlecode.com/svn/trunk/</connection>
+		<connection>scm:svn:https://open-delta.googlecode.com/svn/trunk/</connection>
 		<developerConnection>scm:svn:https://open-delta.googlecode.com/svn/trunk/</developerConnection>
-		<url>http://code.google.com/p/open-delta/</url>
+		<url>https://code.google.com/p/open-delta/</url>
 	</scm>
 	<build>
 		<resources>
@@ -82,8 +82,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>2.0.2</version>
 				<configuration>
-					<source>1.6</source>
-					<target>1.6</target>
+					<source>1.8</source>
+					<target>1.8</target>
 					<encoding>Cp1252</encoding>
 				</configuration>
 			</plugin>
@@ -252,6 +252,13 @@
 			<version>2.0.05</version>
 			<type>jar</type>
 			<scope>compile</scope>
+		</dependency>
+		<dependency>
+			<groupId>javax.jnlp</groupId>
+			<artifactId>jnlp</artifactId>
+			<version>1.6</version>
+			<scope>system</scope>
+			<systemPath>/usr/share/icedtea-web/javaws.jar</systemPath>
 		</dependency>
 		<dependency>
 			<groupId>com.l2fprod.common</groupId>

--- a/intkey/pom.xml
+++ b/intkey/pom.xml
@@ -201,6 +201,10 @@
 		</pluginManagement>
 	</build>
 
+	<properties>
+        	<local.jnlp.path>/usr/share/icedtea-web/javaws.jar</local.jnlp.path>
+	</properties>
+
 	<dependencies>
 		<dependency>
 			<groupId>au.org.ala.delta</groupId>
@@ -258,7 +262,7 @@
 			<artifactId>jnlp</artifactId>
 			<version>1.6</version>
 			<scope>system</scope>
-			<systemPath>/usr/share/icedtea-web/javaws.jar</systemPath>
+			<systemPath>${local.jnlp.path}</systemPath>
 		</dependency>
 		<dependency>
 			<groupId>com.l2fprod.common</groupId>

--- a/key/pom.xml
+++ b/key/pom.xml
@@ -1,5 +1,5 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="https://maven.apache.org/POM/4.0.0" xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="https://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>au.org.ala.delta</groupId>
 	<artifactId>key</artifactId>
@@ -7,9 +7,9 @@
 	<name>KEY</name>
 
 	<scm>
-		<connection>scm:svn:http://open-delta.googlecode.com/svn/trunk/</connection>
+		<connection>scm:svn:https://open-delta.googlecode.com/svn/trunk/</connection>
 		<developerConnection>scm:svn:https://open-delta.googlecode.com/svn/trunk/</developerConnection>
-		<url>http://code.google.com/p/open-delta/</url>
+		<url>https://code.google.com/p/open-delta/</url>
 	</scm>
 	<build>
 		<plugins>
@@ -54,8 +54,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>2.0.2</version>
 				<configuration>
-					<source>1.6</source>
-					<target>1.6</target>
+					<source>1.8</source>
+					<target>1.8</target>
 					<encoding>Cp1252</encoding>
 				</configuration>
 			</plugin>

--- a/pom.xml
+++ b/pom.xml
@@ -1,5 +1,5 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="https://maven.apache.org/POM/4.0.0" xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="https://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>au.org.ala.delta</groupId>
 	<artifactId>open-delta</artifactId>
@@ -7,9 +7,9 @@
 	<name>Open DELTA suite</name>
 	<packaging>pom</packaging>
 	<scm>
-		<connection>scm:svn:http://open-delta.googlecode.com/svn/trunk/</connection>
+		<connection>scm:svn:https://open-delta.googlecode.com/svn/trunk/</connection>
 		<developerConnection>scm:svn:https://open-delta.googlecode.com/svn/trunk/</developerConnection>
-		<url>http://code.google.com/p/open-delta/</url>
+		<url>https://code.google.com/p/open-delta/</url>
 	</scm>
 
 	<modules>

--- a/ui-common/pom.xml
+++ b/ui-common/pom.xml
@@ -140,6 +140,10 @@
 		</pluginManagement>
 	</build>
 
+	<properties>
+        	<local.jnlp.path>/usr/share/icedtea-web/javaws.jar</local.jnlp.path>
+	</properties>
+
 	<dependencies>
 		<dependency>
 			<groupId>au.org.ala.delta</groupId>
@@ -162,7 +166,7 @@
 			<artifactId>jnlp</artifactId>
 			<version>1.6</version>
 			<scope>system</scope>
-			<systemPath>/usr/share/icedtea-web/javaws.jar</systemPath>
+			<systemPath>${local.jnlp.path}</systemPath>
 		</dependency>
 		<dependency>
 			<groupId>commons-lang</groupId>

--- a/ui-common/pom.xml
+++ b/ui-common/pom.xml
@@ -1,5 +1,5 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="https://maven.apache.org/POM/4.0.0" xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="https://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>au.org.ala.delta</groupId>
 	<artifactId>delta-ui-common</artifactId>
@@ -10,16 +10,16 @@
 		<repository>
 			<id>maven2-repository.dev.java.net</id>
 			<name>Java.net Repository for Maven</name>
-			<url>http://download.java.net/maven/2/</url>
+			<url>https://download.java.net/maven/2/</url>
 			<layout>default</layout>
 		</repository>
 	</repositories>
 
 
 	<scm>
-		<connection>scm:svn:http://open-delta.googlecode.com/svn/trunk/</connection>
+		<connection>scm:svn:https://open-delta.googlecode.com/svn/trunk/</connection>
 		<developerConnection>scm:svn:https://open-delta.googlecode.com/svn/trunk/</developerConnection>
-		<url>http://code.google.com/p/open-delta/</url>
+		<url>https://code.google.com/p/open-delta/</url>
 	</scm>
 	<build>
 	
@@ -40,8 +40,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>2.0.2</version>
 				<configuration>
-					<source>1.6</source>
-					<target>1.6</target>
+					<source>1.8</source>
+					<target>1.8</target>
 					<encoding>Cp1252</encoding>
 				</configuration>
 			</plugin>
@@ -156,6 +156,13 @@
 			<artifactId>javahelp</artifactId>
 			<version>2.0.05</version>
 			<type>jar</type>
+		</dependency>
+		<dependency>
+			<groupId>javax.jnlp</groupId>
+			<artifactId>jnlp</artifactId>
+			<version>1.6</version>
+			<scope>system</scope>
+			<systemPath>/usr/share/icedtea-web/javaws.jar</systemPath>
 		</dependency>
 		<dependency>
 			<groupId>commons-lang</groupId>

